apps/definition_load_persist/test/load/persist_test.exs

```diff
@@ -29,17 +29,13 @@ defmodule Load.PersistTest do
           id: "load1",
           dataset_id: "ds1",
           subset_id: "joe",
-          source: "topic",
-          destination: "table",
-          schema: [
-            %Dictionary.Type.String{name: "name"},
-            %Dictionary.Type.Integer{name: "age"}
-          ]
+          source: %Source.Fake{},
+          destination: "table"
         )
 
-      json = Jason.encode!(persist)
+      {:ok, serialized} = Brook.Serializer.serialize(persist)
```

Comment:
Reading the name of this test, I'd expect it to be testing something other than brook serialization of the persist struct. I'm not sure if the name is just misleading or if we should be using the store? It seems like json decoding/encoding or even using the Brook serialization is more testing Brook than it is testing something about persist.

Reply:
since brook serializer is a protocol, this validates that each custom struct maintains that contract across versions.